libs-scala/ports/src/main/scala/com/digitalasset/ports/PortFiles.scala

```diff
@@ -36,9 +36,11 @@ object PortFiles {
     }
 
   private def writeUnsafe(path: Path, port: Port): Unit = {
-    import java.nio.file.StandardOpenOption.CREATE_NEW
+    if (Files.exists(path)) throw new FileAlreadyExistsException(path.toString())
```

Comment:
This is racy. The file can get created between this check and the move. We need to set flags on the move to file if the file already exists and catch the exception.

Reply:
I couldn't find any files on the `move` command for the check. But let's see if there is another way.

Reply:
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#move(java.nio.file.Path,%20java.nio.file.Path,%20java.nio.file.CopyOption...) claims that it will fail by default if it already exists so it looks like we already get what we need.

Reply:
oh perfect.